common/scala/src/main/scala/whisk/core/entity/MemoryLimit.scala

```diff
@@ -41,12 +44,14 @@ import whisk.core.entity.size.SizeInt
 protected[entity] class MemoryLimit private (val megabytes: Int) extends AnyVal {}
 
 protected[core] object MemoryLimit extends ArgNormalizer[MemoryLimit] {
-  protected[core] val MIN_MEMORY = 128 MB
-  protected[core] val MAX_MEMORY = 512 MB
-  protected[core] val STD_MEMORY = 256 MB
+  private val memoryConfig = loadConfigOrThrow[MemoryLimitConfig]("whisk.memorylimit")
```

Comment:
If #3150 gets merged first, let's add "whisk.memorylimit" into the PureConfigKeys singleton.